src/EventHub/EventHub.Autorest/custom/Set-AzEventHubNamespace.ps1

```diff
@@ -0,0 +1,345 @@
+        [System.String]
+        # The name of EventHub Entity.
+        ${Name},
+
+        [Parameter(ParameterSetName = 'SetExpanded', Mandatory, HelpMessage = "The name of EventHub namespace.")]
+        [Microsoft.Azure.PowerShell.Cmdlets.EventHub.Category('Path')]
+        [System.String]
+        # The name of EventHub namespace
+        ${NamespaceName},
+
+        [Parameter(ParameterSetName = 'SetExpanded', Mandatory, HelpMessage = "The name of the resource group. The name is case insensitive.")]
+        [Microsoft.Azure.PowerShell.Cmdlets.EventHub.Category('Path')]
+        [System.String]
+        # The name of the resource group.
+        # The name is case insensitive.
+        ${ResourceGroupName},
+
+        [Parameter(ParameterSetName = 'SetExpanded', HelpMessage = "The ID of the target subscription.")]
+        [Microsoft.Azure.PowerShell.Cmdlets.EventHub.Category('Path')]
+        [Microsoft.Azure.PowerShell.Cmdlets.EventHub.Runtime.DefaultInfo(Script = '(Get-AzContext).Subscription.Id')]
+        [System.String]
+        # The ID of the target subscription.
+        ${SubscriptionId},
+
+        [Parameter(HelpMessage = "The Alternate Name of Eventhub Namespace")]
+        [Microsoft.Azure.PowerShell.Cmdlets.EventHub.Category('Path')]
+        [System.String]
+        ${AlternateName},
+
+        [Parameter(HelpMessage = "The ClusterArmId of Eventhub Namespace.")]
+        [Microsoft.Azure.PowerShell.Cmdlets.EventHub.Category('Path')]
+        [System.String]
+        ${ClusterArmId},
+
+        [Parameter(HelpMessage = "DisableLocalAuth")]
+        [Microsoft.Azure.PowerShell.Cmdlets.EventHub.Category('Path')]
+        [System.String]
+        ${DisableLocalAuth},
+
+        [Parameter(HelpMessage = "EncryptionKeySource")]
+        [Microsoft.Azure.PowerShell.Cmdlets.EventHub.Category('Path')]
+        [System.String]
+        ${EncryptionKeySource},
+
+        [Parameter(HelpMessage = "EncryptionKeyVaultProperty.")]
+        [Microsoft.Azure.PowerShell.Cmdlets.EventHub.Category('Path')]
+        [System.String]
+        ${EncryptionKeyVaultProperty},
+
+        [Parameter(HelpMessage = "EncryptionRequireInfrastructureEncryption")]
+        [Microsoft.Azure.PowerShell.Cmdlets.EventHub.Category('Path')]
+        [System.String]
+        ${EncryptionConfig}, #EncryptionConfig
+
+        [Parameter(HelpMessage = "IdentityType of EventHub Namespace.")]
+        [Microsoft.Azure.PowerShell.Cmdlets.EventHub.Category('Path')]
+        [System.String]
+        ${IdentityType},
+
+        [Parameter(HelpMessage = "IdentityUserAssigneIdentity")]
+        [Microsoft.Azure.PowerShell.Cmdlets.EventHub.Category('Path')]
+        [System.Collections.Hashtable]
+        ${IdentityId},#IdentityId
+
+        [Parameter(HelpMessage = "IsAutoInflateEnabled")]
+        [Microsoft.Azure.PowerShell.Cmdlets.EventHub.Category('Path')]
+        [System.String]
+        ${IsAutoInflateEnabled},
```

Comment:
Change "IsAutoInflateEnabled" to "EnableAutoInflate".Type must be switch parameters. Please refer swaggers to know exact types of these parameters.